cnames_active.js

```diff
@@ -3043,6 +3043,7 @@ var cnames_active = {
   "storelocator": "storelocatorjs.github.io",
   "storybook": "apex-loadbalancer.netlify.app", // noCF
   "storybooks": "storybooks.github.io",
+  "stpcloud": "stoeaves.github.io/StpCloud"
```

Comment:
As the CI error notes, you're missing a comma at the end of the line
```suggestion
  "stpcloud": "stoeaves.github.io/StpCloud",
```